app/assets/stylesheets/mobile/login-modal.scss

```diff
@@ -1,21 +1,59 @@
+.static-login {
+  #main-outlet-wrapper,
+  #main-outlet {
+    padding: 0;
+  }
+  .contents.body-page {
+    margin: 0;
+  }
+}
+.login-welcome {
```

Comment:
I can't figure out how to see this element locally... tried with or without the experimental site setting. 

(Wanted to see why we need to set width/height for it.)

Reply:
Those styles are for the welcome page that renders when `login_required` is enabled

![image](https://github.com/user-attachments/assets/0570fca5-7034-41a5-95e7-af5cf33658a4)